### PR TITLE
Update integration test to reflect the conversion of phasedxpow to qiskit rgate

### PR DESCRIPTION
**Pull request by vtomole**

### CODE REVIEW

**Comment by vtomole** [comment]
Will open a PR that re-runs https://github.com/SupertechLabs/qiskit-superstaq/blob/main/examples/qscout_compile.ipynb after SuperstaQ has been deployed.

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def test_qscout_compile(provider: qss.SuperstaQProvider) -> None:
     circuit = qiskit.QuantumCircuit(1)
     circuit.h(0)
     expected = qiskit.QuantumCircuit(2)
-    expected.u(-np.pi / 2, 0, 0, 0)
+    expected.r(np.pi / 2, -np.pi / 2, 0)
```
Comment (richrines1) [inline qiskit_superstaq/daily_integration_test.py:159]:
```suggestion
    expected.r(np.pi / 2, -np.pi / 2)
```

**Comment by vtomole** [comment]
Done. Also update the notebook anyway.

File: qiskit_superstaq/daily_integration_test.py (hunk 1 above)
Comment (vtomole) [reply qiskit_superstaq/daily_integration_test.py]: Wait, i'm getting `TypeError: r() missing 1 required positional argument: 'qubit'` after applying this.

Comment (richrines1) [reply qiskit_superstaq/daily_integration_test.py]: oops my bad! it was right before, i was thinking of qiskit..RGate()

Comment (vtomole) [reply qiskit_superstaq/daily_integration_test.py:159]: Reverted

**Review by richrines1 — APPROVED** [review]
lgtm!

**Pull request merged**